sdk/identity/identity/src/credentials/interactiveBrowserCredential.ts

```diff
@@ -100,6 +83,7 @@ export class InteractiveBrowserCredential implements TokenCredential {
    */
   public getToken(
     scopes: string | string[],
+    //@ts-ignore
```

Comment:
what is being ignored here? If it's `options` not being used, we should prefix it with underscore e.g. `_options?: GetTokenOptions` -- in general I think that's better than removing unused params when they are important to match the API shape.

Reply:
By adding `@ts-ignore` tried to ignore the `noUnusedLocals` tslint error and got it working for `rushx build` command. But for eslint it is failing with error `Do not use "// @ts-ignore" comments because they suppress compilation errors`. I like the idea of prefixing it with underscore  and have committed this change.